src/components/AppFooterSpan.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const AppFooterSpan = styled.span`
+    cursor: pointer;
+    color: black;
+    float: left;
+    padding: 8px 16px;
+    text-decoration: none;
+    transition: background-color .3s;
+    border: 1px solid #ddd;
```

Comment:
colors should be in theme

Reply:
file added by mistake, currently removed